content-resources/src/main/resources/db.properties

```diff
@@ -2,9 +2,9 @@
 
 db.jndi.name=jdbc/OskariPool
 # standalone
-db.url=jdbc:postgresql://localhost:5432/oskaridb
+db.url=jdbc:postgresql://192.168.0.64:5678/hsy_oskaridb
 db.username=postgres
-db.password=admin
+db.password=sergtsoP
```

Comment:
These are propably some internal development db passwords, but lets not change these :)